libraries-2/src/test/java/com/baeldung/jcommander/usagebilling/cli/FetchCurrentChargesCommandUnitTest.java

```diff
@@ -0,0 +1,74 @@
+package com.baeldung.jcommander.usagebilling.cli;
+
+import com.beust.jcommander.JCommander;
+import org.junit.Before;
+import org.junit.Test;
+
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
+
+public class FetchCurrentChargesCommandUnitTest {
+
+    private JCommander jc;
+
+    @Before
+    public void setup() {
+        jc = JCommander
+          .newBuilder()
+          .addObject(new FetchCurrentChargesCommand())
+          .build();
+    }
```

Comment:
This `@Before` method would be better turned into the initializer - e.g.

```
private JCommander jc = JCommander.newBuilder()
          .addObject(new FetchCurrentChargesCommand())
          .build();
```

Note, it's also relatively unnatural to have just a type name at the end of a line, with the static method call on the next. I know it IS convention to break lines at a `.` in fluent calls. It generally looks more like my example than yours.

Reply:
Moved it out into initializer.
About the formatting - 
I'm using the formatter for IntelliJ from the repo, and this is what I get when I apply it (which I thought was odd as well). Right now, I've fixed it manually.

The formatter behaves a bit oddly for some of the scenarios such as builders (as above), static method calls (`UUID.randomUUID().toString()` all goes in separate lines), etc.